MainTel/MainTel-Troubleshooting/MTM2-Troubleshooting/MTM2_recovery.rst

```diff
@@ -0,0 +1,306 @@
+
+.. _`Restart control system`: https://ts-m2gui.lsst.io/error-handling/error-handling.html#lsst-ts-m2gui-error-restart-control-system
+
+
+
+.. admonition:: Information
+
+    **MTM2 Possible States** are Idle, TelemetryOnly (disable), Open LoopLoop, Close Loop. Each state is indicated by an integer from 1 to 4.
+
+    You can check the MTM2 state in Chronograf dashboards `M2 status`_, `MTM2`_ or  `LOVE view`_.
+
+    MTM2 should always be in the Close Loop state (state 4). A GIS interlock prevents TMA movement if MTM2 is not in the Close Loop state. 
+    
+    .. figure:: _static/MTM2_recovery_state_chronograf.png
+        :name: MTM2 state in chronograf
+        :width: 600
+
+
+
+.. _`M2 status`: https://summit-lsp.lsst.codes/chronograf/sources/1/dashboards/296?refresh=Paused&lower=now%28%29%20-%2015m 
+.. is the clock with status 
+
+.. _`MTM2`: https://summit-lsp.lsst.codes/chronograf/sources/1/dashboards/34?refresh=Paused&lower=now%28%29%20-%2015m
+
+.. _`Love view`: https://summit-lsp.lsst.codes/love/uif/view?id=99  
+.. es la imagen que ya esta Monitor M2 hexapod en el punto 7d
+
+.. maybe what is in info block should be in a MTM2 introduction page. But until we have one it can be here.
+
+.. image 
+
+
+.. In Force Balance = enable in LOVE M2 view or Status = 4 in chronograph. 
+.. https://summit-lsp.lsst.codes/chronograf/sources/1/dashboards/231?refresh=Paused&lower=now%28%29%20-%2015m link to chronograph
+
+
+.. note::
+    Any fault in MTM2 is worth ticketing, this is not a system that should reach a fault state easily.
+
+
+.. _MTM2-recovery-Error-Diagnosis:
+
+Error diagnosis
+^^^^^^^^^^^^^^^
+.. This section should provide simple overview of known or suspected causes for the error.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Post screenshots of the error state or relevant tracebacks.
+
+MTM2 CSC goes to fault and exits the Closed Loop control.
+
+
+Follow these instructions to investigate the state of MTM2 and its cause in the Chronograf `MTM2`_ dashboard. Check:
+
+#. **Power Current**, it will indicate when precisely MTM2 left Closed Loop state (here at about 19:38 where the magenta sharply drops).
+
+    .. figure:: _static/MTM2_recovery_power_current.png
+        :name: MTM2 Power current
+        :width: 600
+
+
+#. **TMA Elevation Position and Elevation Angle measured by M2**: to ensure that they are at the same position after leaving Closed Loop (note time displacement in plots).
+
+    .. figure:: _static/MTM2_recovery_elevation_by_M2.png
+        :name: MTM2 Elevation position 
+        :width: 600
+
+#. **Tangent Fault**: which may indicate excessive forces were the cause of the fault
+
+    .. figure:: _static/MTM2_recovery_tangent_fault.png
+        :name: MTM2 Tangent fault
+        :width: 600
+
+
+#. **Log Message**: which may indicate useful information about the cause (can also be read from the CSC logs).
```

Comment:
Consider commenting out the *CSC logs* comment as we currently don't have info on how to do that. In the future we could probably link to how to extract the CSC logs document.